rag_apps/graphrag_knowledge_system/src/search_engine.py

```
@@ -0,0 +1,205 @@
+                )
+                rating = json.loads(raw)
+                if rating.get("relevant") and float(rating.get("relevance_score", 0)) > 0.25:
+                    scored.append(
+                        {
+                            **comm,
+                            "relevance_score": float(rating.get("relevance_score", 0.5)),
+                            "key_points": rating.get("key_points", []),
+                        }
+                    )
+            except Exception:
+                continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)